gulpfile.babel.js

```diff
@@ -0,0 +1,264 @@
+			.pipe(rollup({
+				entry: './src/app/index.js'
+			}))
+			.pipe(babel())
+			.pipe(uglify())
+			.pipe(rename({
+				baseName: 'client',
+				suffix: '.min'
+			}))
+			.pipe(gulp.dest(settings.destinations.prod.scripts));
+	}
+	return gulp.src(settings.sources.scripts)
+		.pipe(rollup({
+			entry: './src/app/index.js'
+		}))
+		.pipe(babel())
+		.pipe(gulp.dest(settings.destinations.dev.scripts));
+}
+
+export function vendorScripts() {
+	const b = browserify(Object.assign({}, browserifyInc.args));
+	settings.sources.externalJs.forEach(dep => b.require(dep));
+	browserifyInc(b, {cacheFile: './.browserify-cache.json'});
+
+	if (isProdBuild()) {
+		return b.bundle()
+			.pipe(source('vendor.min.js'))
+			.pipe(buffer())
+			.pipe(uglify())
+			.pipe(gulp.dest(settings.destinations.prod.libs));
+	}
+	return b.bundle()
+		.pipe(source('vendor.js'))
+		.pipe(buffer())
+		.pipe(gulp.dest(settings.destinations.dev.libs));
+}
+
+export function bundleExternalCSS(done) {
+	const files = require('./package.json').bootstrapKickstart.bundleCSS;
+	if (files.length < 1) {
+		return done();
+	}
+	if (isProdBuild()) {
+		return gulp.src(files.map(path => 'node_modules/' + path))
+			.pipe(cleanCss())
+			.pipe(concat('libs.min.css'))
+			.pipe(gulp.dest(settings.destinations.prod.libs));
+	}
+	return gulp.src(files.map(path => 'node_modules/' + path))
+		.pipe(concat('libs.css'))
+		.pipe(gulp.dest(settings.destinations.dev.libs));
+}
+
+export function copyStaticFiles() {
+	return gulp.src(settings.sources.staticFiles.map(path => 'node_modules/' + path), {base: 'node_modules/'})
+		.pipe(gulp.dest(isProdBuild() ? settings.destinations.prod.libs : settings.destinations.dev.libs));
+}
+
+export function lint() {
+	if (isProdBuild()) {
+		return gulp.src([settings.sources.scripts, './*.js'])
+			.pipe(eslint())
+			.pipe(eslint.format())
+			.pipe(eslint.failAfterError());
+	}
+	return gulp.src([settings.sources.scripts, './*.js'])
+		.pipe(eslint())
+		.pipe(eslint.format())
+		.pipe(eslint.failAfterError())
+		.on('error', onError);
+}
+
+export function security(cb) {
+	if (isProdBuild()) {
+		nsp({package: path.join(__dirname, '/package.json')}, cb);
+	} else {
+		cb();
+	}
+}
+
+export function html() {
+	if (isProdBuild()) {
+		return gulp.src(settings.sources.markup)
+			.pipe(processhtml())
+			.pipe(gulp.dest(settings.destinations.prod.markup));
+	}
+	return gulp.src(settings.sources.markup)
+		.pipe(changed(settings.destinations.dev.markup))
+		.pipe(gulp.dest(settings.destinations.dev.markup));
+}
+
+export function serve(done) {
+	let baseDir = mainDirectories.dev;
+	if (isProdBuild()) {
+		baseDir = mainDirectories.dist;
+	}
+	server.init({
+		server: {
+			baseDir
+		}
+	});
+	done();
+}
+
+function reload(done) {
+	server.reload();
+	done();
+}
+
+export function watch() {
+	gulp.watch(settings.sources.scripts,
+		gulp.parallel(
+			gulp.series(clientScripts, reload),
+			lint
+		)
+	);
+	gulp.watch('./*.js', lint);
```

Comment:
> Discuss CSS Linting?

[stylelint](https://github.com/stylelint/stylelint) is on the roadmap for v4.
See #127 ➡️ Ideas (Prio 3)